css/style.css

```diff
@@ -0,0 +1,288 @@
+}
+
+.car__title {
+  text-align: center;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 110px;
+  margin-bottom: 0;
+  margin-top: 60px;
+}
+
+.car__subtitle {
+  position: relative;
+  bottom: 11px;
+  text-align: center;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 15px;
+  margin-top: 0;
+}
+
+.car {
+  background: url('../IMG/Tesla-Wordmark-Red.png') no-repeat;
+  margin-bottom: 29px;
+  background-size: 100%;
+}
+
+.car__tsl {
+  display: block;
+  margin: 0 auto;
+  position: relative;
+  right: 35px;
+  width: 67%;
+}
+
+.welcome-page {
+  width: 70%;
+  margin: 0 auto;
+  text-align: center;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.welcome-page__1 {
+  display: flex;
+  justify-content: space-between;
+  width: 41%;
+}
+
+.welcome-page__2 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.welcome-page__block {
+  display: flex;
+  margin-left: 14px;
+}
+
+.welcome-page__driver {
+  margin-right: 20px;
+}
+
+.discrip {
+  display: flex;
+  flex-direction: column;
+}
+
+.welcome-page__speed-sec {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+span {
+  font-size: 20px;
+  line-height: 24px;
+  font-style: normal;
+  font-weight: 600;
+}
+
+.welcome-page__speed-km {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+  text-align: left;
+}
+
+.welcome-page__speed-car {
+  padding: 12px 10px 0 0;
+  width: 89%;
+}
+
+.welcome-page__driver_img {
+  margin-top: 10px;
+  margin-right: 9px;
+  width: 89%;
+}
+
+.welcome-page__driver-max {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  padding-right: 8px;
+}
+
+.welcome-page__driver-go {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.welcome-page__button-image {
+  display: block;
+}
+
+.welcome-page__button-image:hover {
+  width: 110%;
+}
+
+.welcome-page__button {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-right: 37px;
+}
+
+.welcome-page__button:hover{
+  height: 105%;
+}
+
+.welcome-page__test-drive {
+  margin-left: 5px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.welcome-page__test-drive:hover {
+  color: #e51937;
+  cursor: pointer;
+}
+
+.welcome-page__footer {
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 12px;
+  margin-top: 10px;
+  background: white;
+  transition-duration: .5s;
+  transition-property: background;
+}
+
+.welcome-page__footer:hover {
+  background: red;
+}
+
+.welcome-page__footer-btn {
+  text-decoration: none;
+  color: #e51937;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  background: white;
+  text-transform: uppercase;
+  transition-duration: .5s,.5s;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)